graph-refresh/base/cronjob.yaml

```diff
@@ -26,34 +26,31 @@ spec:
             - image: "graph-refresh-job:latest"
               name: graph-refresh
               env:
-                - name: "GRAPH_REFRESH_NO_SOLVERS"
+                - name: DEBUG_LEVEL
                   value: "0"
-                - name: "GRAPH_REFRESH_NO_PACKAGE_ANALYZERS"
-                  value: "0"
-                - name: THOTH_GRAPH_REFRESH_EAGER_STOP
-                  # Stop after N scheduled solver runs per new package-version.
-                  value: "200"
+                - name: APP_SCRIPT
+                  value: "app.sh"
+                - name: KAFKA_BOOTSTRAP_SERVERS
+                  valueFrom:
+                    configMapKeyRef:
+                      name: kafka
+                      key: kafka-bootstrap-servers
+                - name: KAFKA_CAFILE
+                  value: "/mnt/secrets/kafka_ca.crt"
+                - name: KNOWLEDGE_GRAPH_HOST
+                  valueFrom:
+                    configMapKeyRef:
+                      name: thoth
+                      key: postgresql-host
                 - name: KUBERNETES_API_URL
                   value: "https://kubernetes.default.svc.cluster.local"
                 - name: KUBERNETES_VERIFY_TLS
                   value: "0"
-                - name: THOTH_USE_ARGO
-                  value: "1"
-                - name: THOTH_MIDDLETIER_NAMESPACE
-                  valueFrom:
-                    configMapKeyRef:
-                      key: middletier-namespace
-                      name: thoth
                 - name: THOTH_INFRA_NAMESPACE
                   valueFrom:
                     configMapKeyRef:
                       key: infra-namespace
                       name: thoth
-                - name: THOTH_DEPLOYMENT_NAME
```

Comment:
isnt this one required by thoth-common for setting up sentry?!

Reply:
Added again sorry